pkg/api/constants/consts.go

```diff
@@ -31,4 +31,6 @@ const (
 	DeletePermission = "delete"
 	// behaviour actions.
 	DetectManifestCollisionPermission = "detectManifestCollision"
+	// Log String keys.
```

Comment:
"Log" make it lower case

Reply:
Sure, will change this in this file.

However, for the other files, would it be alright to keep this casing? For longer comments, it helps with readability.
Here's a reference I got from golang's website: https://go.dev/doc/comment